src/App.tsx

```diff
@@ -17,18 +65,31 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filterBy={filterBy}
+                query={query}
+                onFilterChange={handleChangeFilter}
+                onQueryChange={handleQueryChange}
+                onClearQuery={handleClearQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && <Loader />}
+
+              <TodoList
```

Comment:
Acording to the working version you need to show TodoList only if delay for loader is finished. In your demo link there is a visible first row of a table of a TodoList component. 
You can just add condition 
```suggestion
              {!isLoader && <TodoList your props here /> }
```